### PR TITLE
Mm filename import

### PR DESCRIPTION
**Pull request by mXaln**

### CODE REVIEW

File: scripts/get_books.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+for x in range(0, 66):
+    # gives book name and order (the books are stored out of order in the json)
+
+    # Having to navigate through DATA a bit because the location of English may move as more languages are added
+    # Using English because the list of books may not be complete in other languages
+    # 0 index after cat is for Bible, 1 contains data for obs
+    # 0 index after vers is for ULB, though UDB would also be fine for this case
+    slug = langIdx["vers"][0]["toc"][x]["slug"]
+    name = langIdx["vers"][0]["toc"][x]["title"]
+    print(slug)
```
Comment (natibekele) [inline scripts/get_books.py]: do we want to have this print statement here?

File: scripts/get_books.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    if LANGS[idx]['lc'] == 'en':
+        langIdx = LANGS[idx]
+
+# skip obs for now, loop over all books
+for x in range(0, 66):
+    # gives book name and order (the books are stored out of order in the json)
+
+    # Having to navigate through DATA a bit because the location of English may move as more languages are added
+    # Using English because the list of books may not be complete in other languages
+    # 0 index after cat is for Bible, 1 contains data for obs
```
Comment (natibekele) [inline scripts/get_books.py]: i don't see `cat` do we still need this comment here?

File: scripts/get_books.py (hunk 1 above)
Comment (mXaln) [reply scripts/get_books.py]: @natibekele I think we can leave it here

File: scripts/get_books.py (hunk 2 above)
Comment (mXaln) [reply scripts/get_books.py]: @natibekele I have no idea, these scripts were written by Craig

**Pull request merged**